src/content/structured/get-started/custom-theme.mdx

```diff
@@ -39,9 +44,148 @@ Alternatively, you can include the `ic-theme` component and set the `color` prop
 <ic-theme color="#5c0949"></ic-theme>
 ```
 
-### Components that use the theme colour
+The components that use the 'brand' colour are:
 
 - [Footer](/components/footer)
 - [Hero](/components/hero)
 - [Side navigation](/components/side-navigation)
 - [Top navigation](/components/top-navigation)
+
+### Theme details
+
+<ComponentDetails component="ic-theme" />
+
+## Dark mode
+
+Each component's variants can be displayed using dark mode colours. This creates better colour contrast with backgrounds, borders and text for when components need to be used on dark backgrounds.
```

Comment:
```suggestion
Each component's variants can be displayed using dark mode colours. This creates better colour contrast with backgrounds, borders and text for when components need to be used on dark backgrounds.
```
I'm not clear on what you are trying to communicate in this paragraph? The benefits of dark mode?

Reply:
I guess a section on why dark mode is a thing, what its intended to be used for?